src/main/java/com/microsoftopentechnologies/windowsazurestorage/WAStorageClient.java

```diff
@@ -331,6 +323,10 @@ public static int upload(AbstractBuild<?, ?> build, BuildListener listener,
 
 			listener.getLogger().println(
 					Messages.WAStoragePublisher_uploading());
+			
+			if (manageArtifacts) {
+                expContainerName = "jobs";
```

Comment:
Is it possible to avoid hard-coding this?  If we can't have different container names for different jobs (or just custom container names in general) then we should probably at least make this configurable on a global level.